src/themes/components/spinner.ts

```diff
@@ -0,0 +1,34 @@
+import { cssVar } from '@chakra-ui/theme-tools';
+import { defineStyle, defineStyleConfig } from '@chakra-ui/styled-system';
+
+const $size = cssVar('spinner-size');
```

Comment:
Is there any reason to not use size tokens directly, given that you map them 1 to 1?